engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -96,6 +96,8 @@ def reset(self) -> None:
         super().reset()
 
     def get_tokenizer(self):
+        if "t5" == self.model_type:
+            return self.engine.preprocessor.tokenizer
```

Comment:
Should it be self.model_type.model_type, if model config is passed?

Reply:
why? `model_type` is set in line 50.

Reply:
NIT: We should probably unify this on the engine side so we don't have to do if-else here. Approving for now

Reply:
yes, agree. I initially tried doing that but it turned out to be a relatively bigger change and did not want to do that at the last minute before the release.